include/Gaffer/ValuePlug.h

```diff
@@ -116,6 +116,18 @@ class GAFFER_API ValuePlug : public Plug
 		static void clearCache();
 		//@}
 
+		/// @name Hash cache management
+		/// In addition to the cache of recently computed values, we also
+		/// keep a per-thread cache of recently computed hashes. These functions
+		/// allow for management of that cache.
+		////////////////////////////////////////////////////////////////////
+		//@{
+		static size_t getHashCacheSizeLimit();
+		/// > Note : Limits are applied on a per-thread basis as and
+		/// > when each thread is used to compute a hash.
+		static void setHashCacheSizeLimit( size_t maxEntriesPerThread );
```

Comment:
bit late since this was merged already, but can we have an equivalent to `clearCache()` (ie `clearHashCache()`) and also clear that in the spots we clear the main cache (eg https://github.com/GafferHQ/gaffer/blob/a475f933a1ac517b2bbb3dc922db97d239ed5056/src/GafferScene/Render.cpp#L268)